DataLayer/Context/GameBoardDbContext.cs

```diff
@@ -1,11 +1,27 @@
-﻿using GameBoard.DataLayer.Repositories;
+﻿using System.Linq;
+using System.Threading.Tasks;
+using GameBoard.DataLayer.Entities;
+using GameBoard.DataLayer.Repositories;
 using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore;
 
 namespace GameBoard.DataLayer.Context
 {
-    internal sealed partial class GameBoardDbContext : IdentityDbContext, IGameBoardRepository
+    internal sealed partial class GameBoardDbContext : IdentityDbContext<ApplicationUser>, IGameBoardRepository
     {
+        public DbSet<ApplicationUser> ApplicationUsers { get; set; }
+        public DbSet<Friendship> Friendships { get; set; }
+
+        public Task<string> GetUserIdByUsername(string username)
```

Comment:
Consistent naming: we use `UserName` (or `userName`) everywhere else.

Reply:
Fixed

Reply:
`GetUserIdByUsername` is still wrong.

Reply:
Fixed.